src/host_capabilities/net.rs

```diff
@@ -0,0 +1,18 @@
+use anyhow::{anyhow, Result};
+use serde_json::json;
+
+/// A list of IP addresses
+pub type LookupResponse = Vec<String>;
+
+/// Lookup the addresses for a given hostname via DNS
+pub fn lookup_host(host: &str) -> Result<LookupResponse> {
+    let req = json!(host);
+    let msg = serde_json::to_vec(&req)
+        .map_err(|e| anyhow!("error serializing the validation request: {}", e))?;
+    let response_raw = wapc_guest::host_call("kubewarden", "net", "dns_lookup_host", &msg)
```

Comment:
Do we want to add versioning as we are doing for [verification](https://github.com/kubewarden/policy-sdk-rust/blob/main/src/host_capabilities/verification.rs#L68)? Unlike verification I don't expect `dns_lookup_host` or `oci_manifest` to change, so I am fine to leave it without versioning.

Reply:
yeah, I don't think we need versioning but honestly I don't think it should harm. I'll version them

Reply:
done